spike_main/spike.cc

```diff
@@ -34,6 +34,7 @@ static void help(int exit_code = 1)
 #endif
   fprintf(stderr, "  -h, --help            Print this help message\n");
   fprintf(stderr, "  -H                    Start halted, allowing a debugger to connect\n");
+  fprintf(stderr, "  --cmd=<name>          Read commands from file (use with -d)\n");
```

Comment:
I see there is an comment in help messag but prefer to adding prefix to option to explicit express it is for interactive mode.  "--debug-cmd", "--interactive-cmd" or something like